alphafold3_pytorch/alphafold3.py

```diff
@@ -2339,7 +2354,7 @@ def forward(self, coords: Float['b n 3']) -> Float['b n 3']:
         translation_vector = rearrange(translation_vector, 'b c -> b 1 c')
 
         # Apply rotation and translation
-        augmented_coords = einsum(centered_coords, rotation_matrix, 'b n i, b i j -> b n j') + translation_vector
+        augmented_coords = einsum(centered_coords, rotation_matrix, 'b n i, b j i -> b n j') + translation_vector
```

Comment:
@lucidrains, note that I had to switch the axes `i` and `j` here to correctly rotate `centered_coords`. I believe we are using the `b i j` einsum rotation syntax in other places of the code, and (if they are similar to this use case) they may be incorrect as well.

Reply:
nice, want to check those other places while you are at it?

your tests are failing though

Reply:
Whew... turns out I originally missed of these axes swaps lol when I was copying over the changes from my fork to `main`. Fixed now, and I'll start looking for all other instances of `b n i, b i j` rotations in the code.

Reply:
It looks like I was mistaken. `WeightedRigidAlign` and `CentreRandomAugmentation` are the only places in the code where rotations are applied to coordinates. I believe we are good to go now